front/app/components/StatusModule/index.tsx

```diff
@@ -123,14 +118,14 @@ const StatusModule = ({ votingMethod, phase, project }: StatusModuleProps) => {
                 formatMessage,
               })}
           </Box>
-          {phase && showDate && phase.attributes.end_at && (
+          {showDate && phase.attributes.end_at && (
             <Text>
               {formatMessage(messages.submittedUntil)}{' '}
-              <b>{getLocalisedDateString(phase?.attributes.end_at)}</b>.
+              <b>{getLocalisedDateString(phase.attributes.end_at)}</b>.
             </Text>
           )}
         </>
-        {!isNilOrError(basketCount) && basketCount > 0 && (
+        {typeof basketCount === 'number' && basketCount > 0 && (
```

Comment:
And also this, but it doesn't change anything since we already had the basketCount > 0 part.